model/user/user_test.go

```diff
@@ -253,3 +253,32 @@ func (s *UserTestSuite) TestGetLoginCache() {
 	s.False(valid)
 	s.Nil(u)
 }
+
+func (s *UserTestSuite) TestClearLoginCache() {
```

Comment:
This test should insert 2 users, and verify that only 1 is deleted in the `ClearLoginCache(u, false)` case, and that both are deleted in the `ClearLoginCache(u, true)` case.

Reply:
For ClearLoginCache(nil, true) it now verifies it worked with a representative user.